xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-api/src/main/java/org/xwiki/notifications/internal/rss/DefaultNotificationRSSRenderer.java

```diff
@@ -0,0 +1,183 @@
+import org.xwiki.model.reference.DocumentReference;
+import org.xwiki.model.reference.DocumentReferenceResolver;
+import org.xwiki.notifications.CompositeEvent;
+import org.xwiki.notifications.NotificationException;
+import org.xwiki.notifications.internal.ModelBridge;
+import org.xwiki.notifications.rss.NotificationRSSRenderer;
+import org.xwiki.rendering.block.XDOM;
+import org.xwiki.rendering.renderer.BlockRenderer;
+import org.xwiki.rendering.renderer.printer.DefaultWikiPrinter;
+import org.xwiki.rendering.renderer.printer.WikiPrinter;
+import org.xwiki.script.ScriptContextManager;
+import org.xwiki.template.TemplateManager;
+
+import com.rometools.rome.feed.synd.SyndContent;
+import com.rometools.rome.feed.synd.SyndContentImpl;
+import com.rometools.rome.feed.synd.SyndEntry;
+import com.rometools.rome.feed.synd.SyndEntryImpl;
+import com.rometools.rome.feed.synd.SyndFeed;
+import com.rometools.rome.feed.synd.SyndFeedImpl;
+import com.rometools.rome.feed.synd.SyndPerson;
+import com.rometools.rome.feed.synd.SyndPersonImpl;
+
+/**
+ * This is the default implementation of {@link NotificationRSSRenderer}.
+ *
+ * @since 9.6RC1
+ * @version $Id$
+ */
+@Component
+@Singleton
+public class DefaultNotificationRSSRenderer implements NotificationRSSRenderer
+{
+    /**
+     * The binding name of a composite event when the description of this composite event is rendered in a feed
+     * entry.
+     */
+    public static final String COMPOSITE_EVENT_BUILDING_NAME = "event";
+
+    @Inject
+    private Logger logger;
+
+    @Inject
+    private ContextualLocalizationManager contextualLocalizationManager;
+
+    @Inject
+    private ModelBridge modelBridge;
+
+    @Inject
+    private DocumentReferenceResolver<String> documentReferenceResolver;
+
+    @Inject
+    private TemplateManager templateManager;
+
+    @Inject
+    private ScriptContextManager scriptContextManager;
+
+    @Inject
+    @Named("html/5.0")
+    private BlockRenderer blockRenderer;
+
+    @Override
+    public SyndEntry renderNotification(CompositeEvent eventNotification) throws NotificationException
+    {
+        // We need at least one event in the given CompositeEvent
+        if (eventNotification.getEvents().size() == 0) {
+            throw new NotificationException("Unable to create a RSS entry from an empty composite event.");
+        }
+
+        SyndEntry entry = new SyndEntryImpl();
+        SyndContent entryDescription = new SyndContentImpl();
+
+        // The users contained in the CompositeEvent are already stored in a Set, they are therefore necessarily unique
+        List<SyndPerson> eventAuthors = new ArrayList<SyndPerson>();
+
+        // Convert every author of the CompositeEvent to a SyndPerson and add it to the new entry
+        for (DocumentReference author : eventNotification.getUsers()) {
+            SyndPerson person = new SyndPersonImpl();
+            person.setName(author.getName());
+            eventAuthors.add(person);
```

Comment:
Same author can be added several times? Use an HashSet to avoid that. Unless SyndEntry handle that already?